versioned_docs/version-8.5/components/best-practices/development/writing-good-workers.md

```diff
@@ -126,7 +130,11 @@ ZeebeClient client = ZeebeClient.newClientBuilder()
   .build();
 ```
 
-Or, in Spring Zeebe:
+:::caution
+[Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe) is a community-maintained project.
+:::
```

Comment:
:wrench: We have the same warning just a few lines above. We could drop the second warning here.

Reply:
Much like 🍪s I love ⚠️s. I will drop it 😆